doc/articles/guides/solution-building-single-targetframework.md

```diff
@@ -6,17 +6,71 @@ uid: Build.Solution.TargetFramework-override
+
+            Notes:
+            - You may optionally close the solution before making changes and reload the solution afterwards. This will avoid Visual Studio
+            asking you to reload any projects, it will also ensure that the changes are picked up by Visual Studio, and trigger a restore of
+            the packages.
+            - You may want to unload the platform heads that you are not going to build for. This will ensure that Visual Studio does not
+            try to build them, and will speed up the build process.
+        -->
+
+        <PropertyGroup>
+            <!-- Uncomment each line for each platform that you want to build: -->
+
+            <!-- <OverrideTargetFramework Condition="''!='hint: Windows App Sdk (WinUI)'">net8.0-windows10.0.19041.0</OverrideTargetFramework> -->
+            <!-- <OverrideTargetFramework Condition="''!='hint: Webassembly'">net8.0-browserwasm</OverrideTargetFramework> -->
+            <!-- <OverrideTargetFramework Condition="''!='hint: Desktop'">net8.0-desktop</OverrideTargetFramework> -->
+            <!-- <OverrideTargetFramework Condition="''!='hint: iOS'">net8.0-ios</OverrideTargetFramework> -->
+            <!-- <OverrideTargetFramework Condition="''!='hint: Android'">net8.0-android</OverrideTargetFramework> -->
+            <!-- <OverrideTargetFramework Condition="''!='hint: MacCatalyst'">net8.0-maccatalyst</OverrideTargetFramework> -->
+        </PropertyGroup>
+    </Project>
+    ```
+
+    Make sure to replace `net8.0` and `-windows10.0.19041.0` with the appropriate version from your `.csproj` project.
+
+- You can commit `solution-config.props.sample` to your source control.
+- Next, make a copy of `solution-config.props.sample` to `solution-config.props`. This file is [automatically loaded](https://github.com/unoplatform/uno/blob/71f1d5ab067c0dcfad2f4cccd310e506cdeaf6bf/src/Uno.Sdk/targets/Uno.Import.SolutionConfig.props#L9) by the `Uno.Sdk`.
+- If you're using git, add this `solution-config.props` to your `.gitignore`.
+
+  > [!NOTE]
+  > This avoids altering the target frameworks or your own CI, as well as other clones made by other developers.
 
-## Improve performance using the Uno Platform templates
+- Follow the directives from the file, uncommenting one of the `OverrideTargetFramework` to your choosing.
+- In the project head, as well as any other libraries using the `Sdk="Uno.Sdk"`, add the following block immediately below the last `<TargetFrameworks>` line:
+
+    ```xml
+    <TargetFrameworks Condition=" '$(OverrideTargetFramework)' != '' ">$(OverrideTargetFramework)</TargetFrameworks>
+    ```
+
+- Once done, if you're in Visual Studio 2022, you may need to close and re-open your solution, or otherwise click the reload button. For other IDEs, the projects will reload automatically.
+
+At this point, you'll notice that the list of target frameworks available in the debugger will have reduced to the list you added in `OverrideTargetFramework`.
+
```

Comment:
```suggestion
- Follow the directives from the file, uncommenting one of the `OverrideTargetFramework` to your choosing.
- In the project head, as well as any other libraries using the `Sdk="Uno.Sdk"`, add the following block immediately below the last `<TargetFrameworks>` line:

    ```xml
    <TargetFrameworks Condition=" '$(OverrideTargetFramework)' != '' ">$(OverrideTargetFramework)</TargetFrameworks>
    ```

- Once done, if you're in Visual Studio 2022, you may need to close and re-open your solution or otherwise click the reload button. For other IDEs, the projects will reload automatically.

At this point, you'll notice that the list of target frameworks available in the debugger will have been reduced to the list you added in `OverrideTargetFramework`.

```